README.md

```diff
@@ -28,17 +28,25 @@ inputs:
     default: 'error'
   reporter:
     description: 'Reporter of reviewdog command [github-pr-check,github-check,github-pr-review].'
-    default: 'github-pr-check'
+    default: 'github-pr-review'
```

Comment:
Can you revert this change?

If we want to change it, we should discuss and create a different PR.

Reply:
Oh, never mind. 
It's a doc fix and there are no behavior change.